nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/FetchFileTransfer.java

```diff
@@ -120,6 +122,13 @@
         .required(false)
         .build();
 
+    static final PropertyDescriptor FILE_NOT_FOUND_LOG_LEVEL = new PropertyDescriptor.Builder()
+        .name("Log level when file not found")
```

Comment:
Can we use both .name and .displayName?

Reply:
added.
The reason for not adding initially is to be consistent with FetchFile and with the rest of descriptors in FetchFileTransfer. But I agree with review and adding displayName